src/style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
+  width: fit-content;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  border: 2px solid transparent;
+  box-sizing: border-box;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > .stars__star:last-child,
+.stars--2 > .stars__star:nth-of-type(n+4),
+.stars--3 > .stars__star:nth-of-type(n+3),
+.stars--4 > .stars__star:nth-of-type(n+2),
+.stars--5 > .stars__star:nth-of-type(n) {
+  background-image: url(images/star-active.svg);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
you don't need this styles here, you already set it above

Reply:
It doesn't work correctly